apps/src/sites/code.org/pages/views/workshop_search.js

```diff
@@ -95,12 +92,13 @@ function createNewMarker(latLng, title, infoWindowContent, subject) {
   return marker;
 }
 
-function iconForSubject(subject) {
-  if (subject === SubjectNames.SUBJECT_CSF_201) {
-    return 'https://maps.google.com/mapfiles/kml/paddle/red-stars.png';
-  }
-  return 'https://maps.google.com/mapfiles/kml/paddle/red-blank.png';
-}
+const iconForSubject = subject => ({
```

Comment:
Nit: Can you just do `{` instead of `({`?

Reply:
I did struggle with the syntax here, but the [internet suggested](https://mariusschulz.com/blog/returning-object-literals-from-arrow-functions-in-javascript) that I needed to use parentheses to return an object literal from an arrow function. Happy to adjust to style standards though!

Reply:
Oh, interesting!